style.css

```diff
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  font-family: "Courier New", Courier, monospace;
+  background-image: url('https://static.wixstatic.com/media/c88f3c_80f37c005f9a46209de3525512af4e1d~mv2.gif');
+  background-size: cover;
+}
+
+p {
+  margin: 1px;
+  font-weight: 300;
+  background-color: #fff;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+}
+
+.cards {
+  margin: 50px auto;
+  width: 90%;
+  height: 600px;
+  background-image: url('https://i.pinimg.com/736x/71/84/8f/71848fd22b78ca45b3dd628888f6b866.jpg');
+  background-size: cover;
+  opacity: 0.9;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 15px;
+}
+
+.killer-deck {
+
+  width: 250px;
+  height: 400px;
+  background-image: url('assets/card-image.png');
+  background-size: cover;
+  text-align: center;
+  padding-top: 30px;
+  border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.5);
+  transition: 0.3s;
+}
+
+.weapon-deck {
+  width: 250px;
+  height: 400px;
+  background-image: url('assets/card-image.png');
+  background-size: cover;
+  text-align: center;
+  padding-top: 30px;
+  border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.5);
+  transition: 0.3s;
+}
+
+.room-deck {
+  width: 250px;
+  height: 400px;
+  background-image: url('assets/card-image.png');
+  background-size: cover;
+  text-align: center;
+  padding-top: 30px;
+  border-radius: 16px;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.5);
+  transition: 0.3s;
+}
+
+button {
+  padding: 8px 16px;
+  background: white;
+  border: none;
+  font-size: 20px;
+  border-radius: 8px;
+  display: block;
+  margin: 0 auto;
+  margin-bottom: 40px;
+}
+.button:hover,
+.button:focus {
```

Comment:
Nice how you've written the same css class, but two different psuedo classes like this. Good tip for me!